modules/aws/ec2/variables.tf

```diff
@@ -63,3 +63,9 @@ variable "user_data_replace_on_change" {
   default     = true
   description = "This value indicates whether changes to the `user_data` value triggers a rebuild of the EC2 instance."
 }
+
+variable "custom_ssh_key" {
```

Comment:
I would suggest the variable is called `custom_key_name` or something indicating that this is the _name_ of a key, rather than the contents of one.

Reply:
Done